fares-data-build-tool/create-data
# feat: cfd 464 update start end dates

# Description

-   sets start date and end date correctly

# Testing instructions

-   Add a start and end date and confirm the dates are presented correctly on the saleConfirmation page
-  Once on Test, confirm the matching JSON contains the correct datetimes

# Type of change
